src/view/InputView.js

```diff
@@ -0,0 +1,27 @@
+import { Console } from '@woowacourse/mission-utils';
+
+import MESSAGES from '../constants/messages.js';
+
+const InputView = Object.freeze({
+  async readLine(query) {
+    const inputValue = await Console.readLineAsync(query);
+    return inputValue;
+  },
```

Comment:
```suggestion
  readLine(query) {
    return Console.readLineAsync(query);
  },
```

p4 - 이런식으로 바꿔서 작성해도 동작한다는 사실을 알고계신가요?!
`inputValue`를 별도로 수정하지 않기때문에 이렇게 작성하는게 더 깔끔할 것 같아요!

[return await ~](https://geuni620.github.io/blog/2023/9/1/return-await/#%EC%B0%B8%EA%B3%A0%EC%9E%90%EB%A3%8C)